src/components/structures/MatrixChat.tsx

```diff
@@ -576,8 +576,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 }
                 break;
             case 'logout':
-                CallHandler.instance.hangupAllCalls();
-                if (VideoChannelStore.instance.connected) VideoChannelStore.instance.setDisconnected();
+                LegacyCallHandler.instance.hangupAllCalls();
+                for (const call of CallStore.instance.activeCalls) call.setDisconnected();
```

Comment:
Shouldn't this be `disconnect()` (or whatever is its name)? `setDisconnected()` doesn't kill the widget right, so it might keep recording